src/main/java/com/shopify/mappers/LegacyToFulfillmentOrderMapping.java

```diff
@@ -118,6 +120,8 @@ public static ShopifyFulfillmentPayloadRoot toShopifyFulfillmentPayloadRoot(fina
 					}
 				}
 			}
+			if (lineItemsByFulfillmentOrder.size() < 1)
+				throw new ShopifyUnsupportedActionException(SupportedActions.CREATE_FULFILLMENT);
```

Comment:
I'm fine with leaving this as an exception if you want. it's a little confusing here though because it's not shopify stating that we can't do that. (We actually can if we have the right data passed in here). It's actually the input value that is preventing the action here. My goto usually if i don't want to write my own exception would be to throw an IllegalStateException that throws with the error message of something like: 'You can't create a fulfillment without any line items'. I think this would make things a bit clear to the end user.

Reply:
It's because in the case we got, the empty lineItems are caused by a lack of supported actions. But it does makes sense to change this one into a clearer message to the final user